sessions/js-dom-and-events/demo-start/index.html

```diff
@@ -6,14 +6,12 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>JS Basics: Console Methods</title>
+    <title>JS Basics: Demo</title>
```

Comment:
Is this not the HTML for the JS DOM and Events session?

```suggestion
    <title>JS DOM and Events</title>
```